packages/react/components/link/Link.native.tsx

```
@@ -1,11 +1,11 @@
-import { ComponentName } from '@/components/enumsComponentsName'
-import { Icon } from '@/components/icon'
-import { Spacer, SpacerSize } from '@/components/spacer'
-import { TextLevels } from '@/components/text'
-import { getColorStyle, TrilogyColor } from '@/objects/facets/Color'
-import * as React from 'react'
-import { Linking, Platform, StyleSheet, Text, TouchableOpacity, View } from 'react-native'
-import { LinkPropsNative } from './LinkProps'
+import { ComponentName } from '@/components/enumsComponentsName';
```

ESLint

[error] Extra semicolon.

(semi)